doc/source/mapping.rst

```diff
@@ -26,11 +24,63 @@ Chloropleth Maps
     @savefig world_randomcolors.png width=5in
     world.plot();
 
+Note that in general, any options one can pass to `pyplot <http://matplotlib.org/api/pyplot_api.html>`_ in ``matplotlib`` (or `style options that work for lines <http://matplotlib.org/api/lines_api.html>`_) can be passed to the ``plot()`` method. 
+
+
+Chloropleth Maps
+-----------------
+
+*geopandas* makes it easy to create Chloropleth maps (maps where the color of each shape is based on the value of an associated variable). Simply use the plot command with the ``column`` argument set to the column whose values you want used to assign colors. 
+
+.. ipython:: python
+
     # Plot by GDP per capta
+    world = world[(world.pop_est>0) & (world.name!="Antarctica")]
     world['gdp_per_cap'] = world.gdp_md_est / world.pop_est
     @savefig world_gdp_per_cap.png width=5in
     world.plot(column='gdp_per_cap');
 
 
-Adding Basemaps
+Color Maps
+~~~~~~~~~~
+
+One can also modify the colors used by ``plot`` with the ``cmap`` option (for a full list of colormaps, see the `matplotlib website <http://matplotlib.org/users/colormaps.html>`_):
+
+.. ipython:: python
+
+    @savefig world_gdp_per_cap_red.png width=5in
+    world.plot(column='gdp_per_cap', cmap='OrRd');
+
+
+The way color maps are scaled can also be manipulated with the ``scheme`` option. By default, ``scheme`` is set to 'equal_intervals', but it can also be adjusted to any other `pysal option <http://pysal.org/1.2/library/esda/mapclassify.html>`_, like 'quantiles', 'percentiles', etc.  
+
+.. ipython:: python
+
+    @savefig world_gdp_per_cap_quantiles.png width=5in
+    world.plot(column='gdp_per_cap', cmap='OrRd', scheme='quantiles');
+
+
+Maps with Layers
 -----------------
+
+To overlap different layers, first ensure they share a common CRS (so they will align), then plot.
+
+.. ipython:: python
+    
+    # Look at capitals
+    # Note use of standard `pyplot` line style options 
+    # (points are still "lines" to matplotlib). 
+    @savefig capitals.png width=5in
+    cities.plot(marker='*', color='green', markersize=5);
+
+    # Check crs
+    cities = cities.to_crs(world.crs)
+
+    # Overlay over country outlines
+    #[anyone know how to do this?]
```

Comment:
Two possible options:

```
ax =  world.plot()
cities.plot(ax=ax)
```

or

```
fig, ax = plt.subplots()
ax.set_aspect('equal')
world.plot(ax=ax)
cities.plot(ax=ax)
```

I find the second the cleanest, but the drawback is that you then have to know to set the aspect to equal (which should be done for mapping)